app/views/complaints/_complaint_unlink_modal.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="margin-y-3">
+  <div
+    class="usa-modal"
+    id="unlink-modal-<%= item %>"
+    aria-labelledby="unlink-modal-heading"
+    aria-describedby="unlink-modal-description"
+  >
+    <div class="usa-modal__content">
+      <div class="usa-modal__main">
+        <h2 class="usa-modal__heading" id="unlink-modal-heading">
+          Are you sure you want to unlink this <%= item_type %>?
+        </h2>
+        <div class="usa-prose">
+          <p id="unlink-modal-description">
+            This will remove the <%= item_type %> from this complaint, but will not delete the report from <%= item_system %>
+          </p>
+        </div>
+        <div class="usa-modal__footer">
+          <ul class="usa-button-group">
+            <li class="usa-button-group__item">
+            <%= form_with url: public_send(unlink_path, @complaint.id), method: "delete" do %>
```

Comment:
Preferences on whether we use `public_send(string_path, args)` vs trying to pass through something like `path_uri(args)` itself?

Reply:
i think since this is all done in the view, you should be able to pass the method in directly and not need to mess with public_send.

ie

`unlink_path: unlink_tta_report_path,`

and then just use `unlink_path`

Reply:
cool!  I should have tried that